wordpress/1.11/README.md

```diff
@@ -0,0 +1,239 @@
+
+- Finally, click on `Review & Run` button.
+
+![MySQL Install](img/mysql-install.png)
+
+- Wait for service to be up and running, go to Services section and check the service status.
+- Now, go to [Customize MySQL](#customize-mysql) section.
+
+## Install MySQL using CLI
+
+Create a file called `mysql-config.json` and set the following properties:
+
+```json
+{
+  "service": {
+    "name": "mysql"
+  },
+  "database": {
+    "name": "wordpress",
+    "username": "wp_admin",
+    "password": "password",
+    "root_password": "wordpress"
+  },
+  "networking": {
+    "port": 3306,
+    "host_mode": true
+  }
+}
+```
+
+Then, install MySQL:
+
+```bash
+$ dcos package install mysql --options=mysql-config.json
+By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#community-services
+This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.
+
+```Advanced Installation options notes```
+
+storage / *persistence*: create local persistent volumes for internal storage files to survive across restarts or failures.
+
+storage / persistence / *external*: create external persistent volumes. This allows to use an external storage system such as Amazon EBS, OpenStack Cinder, EMC Isilon, EMC ScaleIO, EMC XtremIO, EMC VMAX and Google Compute Engine persistent storage. *NOTE*: To use external volumes with DC/OS, you MUST enable them during CLI or Advanced installation.
+
+storage / *host_volume*:  if persistence is not selected, this package can use a local volume in the host for storage, like a local directory or an NFS mount. The parameter *host_volume* controls the path in the host in which these volumes will be created, which MUST be the same on all nodes of the cluster.
+
+NOTE: If you didn't select persistence in the storage section, or provided a valid value for *host_volume* on installation,
+YOUR DATA WILL NOT BE SAVED IN ANY WAY.
+
+networking / *port*: This DC/OS service can be accessed from any other application through a NAMED VIP in the format *`service_name.marathon.l4lb.thisdcos.directory:port`*. Check status of the VIP in the *Network* tab of the DC/OS Dashboard (Enterprise DC/OS only).
+
+networking / *external_access*: create an entry in Marathon-LB for accessing the service from outside of the cluster
+
+networking / *external_access_port*: port to be used in Marathon-LB for accessing the service.
+Continue installing? [yes/no] yes
+Installing Marathon app for package [mysql] version [5.7.12-0.3]
+Service installed.
+
+Default login: `admin`/`password`. This username/password combination only applies if you haven't changed the defaults.
+```
+
+## Customize MySQL
+
+- We need to customize our mysql service for further development. I will explain it later.
+- Go to `mysql` service from services menu, click on `Edit` button.
+- Chnage `Container Runtime` from `Docker Engine` to `Universal Container Runtime(UCR)`.
```

Comment:
Testing under 1.11.2, the "Edit" menu off of the services menu just brings you back to the original configuration screens for the service. Where are you finding the screen to change which container runtime is being used?

Also, a little thing: typo for the word "Change"

Reply:
Here, I am talking about editing mysql service after it is up and running. It is the only way currently possible to change container runtime. There is no way to set container runtime when we deploy our service from GUI and CLI. Does that answer your question?

Reply:
Can you explain exactly how you get to this screen? When I click "Edit" I get the original setup menu, not one that lets me edit the containerizer.

Reply:
I deployed mysql service and select it from services. then clicked on `Edit` from right top corner, it is same screen as we can edit any other services, on left side, there are options like Service, Placement, Networking etc.